distribution/docs/src/main/resources/_contents/_configuring/configuring-catalog-provider-contents.adoc

```diff
@@ -0,0 +1,10 @@
+
+==== Reconfiguring ${branding} with a Different Catalog Provider
+
+This scenario describes how to reconfigure ${branding} to use a different catalog provider.
+
+This scenario assumes ${branding} is already running.
+
+. Uninstall Catalog Provider (if installed).
+. Install the new Catalog Provider by installing its feature.
+. Verify ${ddf-branding} is running with the new Provider as its Catalog Provider by going to the *Services* tab in the ${admin-console}.
```

Comment:
This seems to be an old statement. We don't have a Services tab in the current admin UI correct?